src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiLaunchSpecExtensionsSpecification.java

```diff
@@ -0,0 +1,127 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiLaunchSpecExtensionsSpecification implements IPartialUpdateEntity {
```

Comment:
publicSettings field is missing (refer - https://docs.spot.io/api/#operation/azureStatefulNodeCreate)

Reply:
on hold for clarification, will fix as part of document review